docs/samples/deployments/spark/index.md

```diff
@@ -7,141 +7,107 @@ The following sample uses [Azure Databricks](https://docs.microsoft.com/en-us/az
 
 ## Pre-requisites
 
-Provision and setup the required infrastrucutre
+If you do not have an instance of Azure Databricks, follow through with the following steps to provision and setup the required infrastrucutre.
 
-### Azure Storage
+If you do have a Databricks workspace and a cluster you wish to configure to run Presidio, jump over to the [Configure an existing cluster](#Configure-an-existing-cluster) section.
 
-Provision an Azure storage account using the following script.
+### Deploy Infrastructure
```

Comment:
Not directly related to this PR, but there is no link for the databricks notebook. We had a few issues opened by users trying to implement this in Spark, and they couldn't find the notebook you created ([This for example](https://github.com/microsoft/presidio/discussions/669#discussioncomment-715677)). 

Maybe we can add a code snippet with the pandas_udf implementation, or direct users to the implementation from this doc? It's currently focused on the devops aspects which is great, but some users just want to see how the spark code looks like and they don't need to set everything up (or they don't use databricks).